lib/build/bundle.js

```diff
@@ -0,0 +1,63 @@
+var assign = require("lodash").assign;
+var minify = require("../stream/minify");
+var devBundle = require("../stream/dev_bundle");
```

Comment:
@matthewp ok, I'll change this.... if `filter` is set to `node_modules/**/*` when `--dev` is passed,  should we throw if no `filter` value is passed for those using the API instead of the CLI?